### ISSUE TITLE
Staging Review finding: Focus not managed on success

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#125856]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** MHV-Secure Messaging
**Product:** Secure Messaging
**Feature:** Medical Renewal Via Secure Messaging

## Findings details
**VA.gov Experience Standard - issue:** [User does not receive appropriate notification from VA.gov or VA Mobile App about the status of their request.](https://design.va.gov/about/experience-standards/#user-receives-appropriate-notification-from-the-va-or-vagov-about-the-status-of-their-request)
**VA.gov Experience Standard - category:** [Credibility](https://design.va.gov/about/experience-standards/#category-credibility)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

When the user sends a message, and is navigated back to the Medications page, focus is not managed - the user does not receive a confirmation of a successfully sent message, unless they navigate to the success alert.

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="912" height="330" alt="confirmation" src="https://github.com/user-attachments/assets/2d0af402-8680-4f90-aa6f-59d529a0d08e" />

</details>

## Recommended action

Ensure focus is managed correctly and that focus is placed on the success alert. 

OR, because the status message is the result of user action, actively announce the success alert using an `aria-live` region. 

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 2.4.3 Focus Order - Level A](https://www.w3.org/TR/WCAG22/#focus-order)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
